packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -81,6 +82,11 @@ class AppBarTheme with Diagnosticable {
   /// If null, the value is adapted to current [TargetPlatform].
   final bool? centerTitle;
 
+  /// Default value for [AppBar.titleSpacing].
+  ///
+  /// If null, [AppBar] uses default value of [NavigationToolbar.kMiddleSpacing].
+  final double titleSpacing;
```

Comment:
NNBD has made it in here :) 
```suggestion
  final double? titleSpacing;
```